app/screens/test_screen/test_screen.js

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import {SafeAreaView, StatusBar, View, Text, Button} from 'react-native';
+
+import EventEmitter from 'mattermost-redux/utils/event_emitter';
+
+import {NavigationTypes} from 'app/constants';
+import ephemeralStore from 'app/store/ephemeral_store';
+
+export default class TestScreen extends PureComponent {
+    static propTypes = {
+        actions: PropTypes.shape({
```

Comment:
0/5 on this right now, but I've been wondering if we should drop the `actions` part of `this.props.actions.loadMe` and just pass the action as `this.props.loadMe`. It would save some typing and I don't know if Realm supports the same shorthand as Redux, but it would be nice to be able to write
```
const mapDispatchToProps = {
    loadMe,
};

export default connect(null, mapDispatchToProps)(MyComponent);
```
instead of having to call `bindActionCreators` manually

Reply:
will try it

Reply:
This is now possible to do.

Did not change the test screen, cause this one shows how you can combine, redux and realm actions